### ISSUE TITLE
ArmA's BattlEye anti-cheat is automatically global banning users of Mumble?!

### ISSUE THREAD

**Issue by agentx5** [mumble-voip/mumble#1567]
Saw this article, and there's a lot of speculation at this is happening to push TeamSpeak over Mumble.
http://www.reddit.com/r/arma/comments/22ge5a/understanding_battleye_false_positives/

I've emailed BattlEye support for more information, but they won't confirm nor deny anything.  They just point fingers (so to speak) back at the application (i.e.: they blame Mumble for any false positives)

Can we please have somebody officially involved with Mumble take a look into this issue and get information published on it?   It's crazy that TeamSpeak and FRAPS have no issue with their overlays, yet people get banned for using Mumble's overlay?

I don't want to use teamspeak, I want to use Mumble!

I can provide a DxDiag on request from Mumble devs and specific version #'s of BattlEye and ArmA 2 (and its mods' version #'s).  The version of Mumble I currently have installed is 1.2.8

Please RSVP, this affects way more than just me.  Everyone in my gaming community uses Mumble, yet I'll be honest most of us are afraid to go on other servers with Mumble running that we might risk getting banned with no real option to appeal such a ban.  We use Mumble in all of the other games we play, and while many of us were skeptical to switch over from Ventrilo at first, we have really enjoyed the switch.

Thank you in advance for your time,

Neil

**Comment by Dwarden** [comment]
dear sir, your claims and assumptions about BattlEye false positives are wrong, 
especially while using nearly year old obsolete and incorrect Reddit article from known cheat author
I'm using Mumble for years and never got banned by our anti-cheat for it or it's overlay 
note: we aware no system is perfect but we work hard to make sure false positives don't happen

**Comment by agentx5** [comment]
Thank you for your reply Dwarden! 

I'm just honestly trying to get to the bottom of this. If it's rumors that are false, then somewhere there needs to be an official note on either Mumble's compatibility list or (ideally) on BattleEye's website's support or info pages.  It would be in everyone's interests (except for the cheaters of course) to squash such rumors if they are indeed false.  And begging your pardon, but I have no way of knowing that you are who you claim to be, much like the "known cheat author" you have labeled that Reddit's article author as.  (and you could totally be right, but put yourself in my shoes for a moment and realize why I might be skeptical of both what you and him claim)

For example, I'd feel much safer and these rumors could be put to rest if this quote from BattlEye's website was amended to include mention of compatible voicechat VoIP program overlays that won't trigger a false positive.

"No one is banned for using non-hack programs (like Fraps, overlays, etc.), picking up or using hacked in-game items, weapons or vehicles, being on a server at the same time as a cheater, or other passive non-cheating activity." -- http://battleye.com/support.html
^ update with a list of approved overlays

If you have the capacity to affect such a change, then I welcome it and ask that you would.  If not, then could you help me get in contact with the individual(s) who could?

I, and the rest of our Sector-Thirteen gaming community, welcome further feedback.  You are also more than welcome to join us on Mumble at anytime.

In short, "proof is in the pudding" as they say.  It's not compatible until I see it written that Mumble and BattlEye are compatible (not triggering false positives) on the official website(s) information.

**Comment by agentx5** [comment]
As side note, I'm well aware of social engineering that cheaters have used in the past against VAC (i.e.: http://www.reddit.com/r/gaming/comments/1y70ej/valve_vac_and_trust )

I just want to reinforce the trust between the players who use BattlEye in ArmA and also use the Mumble overlay in-game. To get there it's going to take getting something in writing on the official website that the two are ok together, then if those are just false rumors then they can be proven false.

I hope I'm making sense...

**Comment by agentx5** [comment]
I have received an official reply from BattlEye support.  I shall quote, and then be closing the topic.

"We do not and never have banned for Mumble. Please do not believe what random people (especially hack creators) claim about BE on the internet.

Regards,
Bastian Suter
BattlEye Innovations"

That said, it would be beneficial to have it mentioned on the actual information on the website.  Further testing is welcome if you wish, but I am closing this topic.
